src/main/java/org/datavyu/plugins/NativeMediaPlayer.java

```diff
@@ -37,10 +34,10 @@
     private EventQueueThread eventLoop = new EventQueueThread();
     private final Lock disposeLock = new ReentrantLock();
     private boolean isDisposed = false;
-    private double startTime = 0.0;
-    private double stopTime = Double.POSITIVE_INFINITY;
-    private boolean isStartTimeUpdated = false;
-    private boolean isStopTimeSet = false;
+    protected double startTime = 0.0;
```

Comment:
Only StartTimeUpdated needs to be protected because it is used in the FFmpeg media player to switch to the accurate flag when the start time is updated.
Others don't have to be protected now, but I anticipated boundary checks implementations, I will put them back to private since we might not use these variables to implement the boundary check feature.